api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentGroupController.java

```diff
@@ -327,6 +330,67 @@ public PaymentGroupController(PaymentGroupService paymentGroupService, PaymentGr
         return new ResponseEntity<>(paymentDtoMapper.toBulkScanPaymentDto(newPayment, paymentGroupReference), HttpStatus.CREATED);
     }
 
+
+    @ApiOperation(value = "Create telephony card payment in Payment Group", notes = "Create telephony card payment in Payment Group")
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = "Payment created"),
+        @ApiResponse(code = 400, message = "Payment creation failed"),
+        @ApiResponse(code = 422, message = "Invalid or missing attribute")
+    })
+    @PostMapping(value = "/payment-groups/{payment-group-reference}/telephony-card-payments")
+    @ResponseBody
+    @Transactional
+    public ResponseEntity<TelephonyDto> createCardPaymentForPCIPALAntenna(
```

Comment:
I suggest changing method name. We should not use 3rd party names in code. I suggest createTelephonyCardPayment as a method name

Reply:
Fixed.